packages/zipkin-instrumentation-axios/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "zipkin-js-instrumentation-axios",
+  "version": "0.2.0",
+  "description": "Library to instrument the axios HTTP-client to Support zipkin.",
+  "main": "lib/index.js",
+  "scripts": {
+    "test": "mocha --compilers js:@babel/register --exit",
+    "build": "babel src --out-dir lib",
+    "prepublish": "babel src --out-dir lib"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/geekeren/zipkin-js-instrumentation-axios.git"
+  },
+  "keywords": [
+    "zipkin",
+    "open-trace",
```

Comment:
I think this keyword `open-trace` is misleading as this repo does not support OT.